src/controllers/events.cr

```diff
@@ -698,6 +698,17 @@ class Events < Application
     render json: visitors
   end
 
+  # example route: /ext_metadata?field_name=colour&value=blue
+  get("ext_metadata", :event_metadata) do
+    field_name = route_params["field_name"]
+    value = route_params["value"]
```

Comment:
These are `query_params`.
Route params would be defined like this `/extension_metadata/:field_name/:value`.

Reply:
The `params` object attached to the controller instance in action-controller is both the `query_params` merged with the `route_params`